astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -232,11 +233,16 @@ def delete_redshift_cluster_callable() -> None:
         trigger_rule="all_done",
     )
 
-    delete_redshift_cluster = PythonOperator(
+    # [START howto_operator_redshift_delete_cluster_async]
+    delete_redshift_cluster = RedshiftDeleteClusterOperatorAsync(
         task_id="delete_redshift_cluster",
-        python_callable=delete_redshift_cluster_callable,
```

Comment:
If not other places we are using `delete_redshift_cluster_callable` please remove it.

Reply:
Removed.